resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -307,6 +307,12 @@ private[spark] class Client(
       }
     }
     appContext.setUnmanagedAM(isClientUnmanagedAMEnabled)
+
+    sparkConf.get(APPLICATION_PRIORITY) match {
+      case Some(v) => appContext.setPriority(Priority.newInstance(v))
+      case None => logDebug(s"${APPLICATION_PRIORITY.key} is not set. " +
```

Comment:
This debug message is not really useful. Just do `sparkConf.get(...).foreach`.

Reply:
done, thanks